build-tools/scripts/BuildEverything.mk

```diff
@@ -11,7 +11,7 @@ GIT_COMMIT        = $(shell LANG=C git log --no-color --first-parent -n1 --prett
 -num-commits-since-version-change = $(shell LANG=C git log $(-commit-of-last-version-change)..HEAD --oneline 2>/dev/null | wc -l | sed 's/ //g')
 
 ZIP_OUTPUT_BASENAME   = oss-xamarin.android_v$(PRODUCT_VERSION).$(-num-commits-since-version-change)_$(OS)-$(OS_ARCH)_$(GIT_BRANCH)_$(GIT_COMMIT)
-ZIP_OUTPUT            = $(ZIP_OUTPUT_BASENAME).zip
+ZIP_OUTPUT            = $(ZIP_OUTPUT_BASENAME).tar.bz2
```

Comment:
I already changed the Jenkins job config to look for any extension, so it won't break the job.

The real space saving ought to be fairly minor (around 15 meg iirc), but because the --exclude logic isn't working properly with `zip`, it ends up being closer to 250 meg by not including both the bundle-* zips and their contents.

The main reason for this change is that `zip` is not considered a valid archive type by dpkg-buildpackage, so users will need to recompress to .tar.foo before being able to build packages